cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/artifact/ArtifactInspector.java

```diff
@@ -381,14 +381,18 @@ private String getPluginName(Class<?> cls) {
    * not specify any {@link io.cdap.cdap.api.annotation.Requirements} then the {@link Requirements} will be empty.
    */
   @VisibleForTesting
-  Requirements getPluginRequirements(Class<?> cls) {
+  Requirements getRequirementsAnnotation(Class<?> cls) {
```

Comment:
This method actually returns the `Requirements` class, not the annotation, so better call it `getArtifactRequirements()`

Reply:
done.